src/IPHlpApi/IPHlpApi+MIB_TCPROW_OWNER_PID.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+#if !NETSTANDARD1_1
+    using System.Net.NetworkInformation;
+#endif
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="MIB_TCPROW_OWNER_PID"/> nested type.
+    /// </content>
+    public static partial class IPHlpApi
+    {
+        /// <summary>
+        /// The MIB_TCPROW_OWNER_PID structure contains information that describes an IP v4 TCP connection
+        /// with IP v4 addresses, ports used by the TCP connection, and the specific process ID (PID)
+        /// associated with connection.
+        /// </summary>
+        /// <seealso href="http://msdn2.microsoft.com/en-us/library/aa366913.aspx"/>
+        [StructLayout(LayoutKind.Sequential)]
```

Comment:
nit: I just learned that this attribute represents the C# default. So can we drop them? See #469